library/swift/src/filters/RequestFilter.swift

```diff
@@ -17,7 +26,7 @@ public protocol RequestFilter: Filter {
   ///
   /// - returns: The header status containing headers with which to continue or buffer.
   func onRequestHeaders(_ headers: RequestHeaders, endStream: Bool)
-    -> FilterHeaderStatus<RequestHeaders>
+    -> UInt32
```

Comment:
Is this the intended end state for this interface? I assume not since it doesn't actually allow the filter to mutate the headers (the existing interface requires the filter to return a status containing headers, which also has the benefit of ensuring the filter cannot mutate the headers after the function returns)

Reply:
Fixed in other PR.